glslc/src/file_compiler.cc

```diff
@@ -168,8 +170,30 @@ bool FileCompiler::EmitCompiledResult(
     }
   }
 
+  // Should write the result as a C-style initializer list if:
+  //  1) the result is SPIRV binary code;
+  //  2) the compiler is set to emit result as a C-style initializer list;
+  //  3) the compiler is not dumping dependency info as compilation output.
+  const bool should_emit_spirv_binary_as_c_initializer_list =
```

Comment:
The above comment just repeats the (already readable) code below it.  I'd rather see an explanation of why we must defend against incompatible flags this far down the call stack.